### DISCUSSION TITLE
Recommended Vercel/GitHub gitflow

### DISCUSSION THREAD

**Discussion by mpicod** [vercel/vercel#4967] [Help] [answered]
Hi,

I'm building a website for a client with Gatsby/Prismic and I'm using the Github integration to deploy my code. 
On the other side we use Prismic webhook to redeploy on CMS update. 

I was wondering if we had somewhere some best practices for managing branches and deployements, as well as naming convention, as it's getting a bit confused in my Vercel Deployement Dashboard with every push being deployed. 
For example my client would like to see only Prod and Staging deployements but is currently seeing every push on every branches. 

Thanks !

**Comment by styfle** [comment, marked as answer]
When you have the [GitHub Integration](https://vercel.com/github) installed, each PR will generate a Preview URL.

You can share that URL with your customer/client and it will stay updated as you push changes to the branch so they can review the changes visually.

For example:

```
https://projectname-git-branchname.username.vercel.app
```
